packages/build-utils/src/lambda.ts

```diff
@@ -123,7 +126,9 @@ export class Lambda {
         '"regions" is not a string Array'
       );
     }
+
     this.type = 'Lambda';
+    this.operationType = operationType;
```

Comment:
Should this default to API when undefined?

Reply:
We could, but the current plan was to default it only for uploading to deployment metadata: https://github.com/vercel/api/pull/15237/files#diff-ce539d6a5c1ccfc0866021b6133828ca803f22791586c1e79037bbcab134cbabR130-R135

I liked keeping the original value (`undefined`, potentially) up until it was displayed. That way, if we use this for anything else, we can know if it was intentionally set or not.